src/InputField/index.js

```diff
@@ -9,7 +9,6 @@ import { theme } from '../theme'
 class InputField extends React.Component {
     state = {
         focus: this.props.focus,
```

Comment:
This could arguable be removed as well. But I think for now it's fine as it's used for styling purposes our apps shouldn't care about